### PR TITLE
Drone Evolution T3: The Reaper

### PR DESCRIPTION
**Pull request by Geevies**
# About the pull request

Adds a new T3, the Reaper (with sprites from our very own Wei/sleepynecrons), which evolves from the Carrier. It's a support combatant with a focus on taking single targets out of action for a long time.

Its abilities are:

- Improved tail stabs, called Claw Strikes. It has an even further extended range of 3 tiles, one more than standard, and the strikes drags your target toward you like an oppressor hook.
- Building xeno special structures, like recovery nodes, hive clusters, etc.
- Planting standard weeds.
- Creating carrier traps.
- A new ability called Plasma Strike, which makes your next slash/strike inject your target with egg plasma and hemocatalene, the latter providing the target with increased blood regeneration, brute damage healing, and toxin damage.
- A new ability called Prime Claw Toss, which makes your next slash/strike throw your target two tiles away from you.
- Passive aura that gives moderate (hivelord level) Warding, Frenzy, and Regen.

# Explain why it's good for the game

The drone evo line has forever lacked a T3, which is a shame, as it's one of my favourite evo lines. The goal of this caste is to provide a fun support/combat blend, while also providing a unique challenge to the medical teams on the marine side.

The egg plasma is powerful, but is offset by the hemocatalene to not instantly kill the target. If the marine is sent for dialysis soon enough, both chemicals can be purged within seconds.

# Testing Photographs and Procedure

I'll post these after the first couple testmerges.

# Changelog

:cl: Geeves, sleepynecrons
add: Added a new T3 caste for the drone line called the Reaper. It evolves from the carrier and has a support-combat focus.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/abilities/reaper/reaper_powers.dm [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		return
+	var/datum/behavior_delegate/base_reaper/behavior = xeno.behavior_delegate
+	if(istype(behavior))
+		// In case slash has already landed
+		if(!behavior.egg_plasma_primed)
+			return
+		behavior.egg_plasma_primed = FALSE
+		behavior.remove_egg_plasma_slash_overlay()
+
+	to_chat(xeno, SPAN_XENODANGER("You have waited too long, your slash will no inject egg plasma!"))
```
Comment (stanalbatross) [inline code/modules/mob/living/carbon/xenomorph/abilities/reaper/reaper_powers.dm:32]:
```suggestion
	to_chat(xeno, SPAN_XENODANGER("You have waited too long, your slash will no longer inject egg plasma!"))
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]
+ more detailed changelog needed

File: code/modules/reagents/chemistry_reagents/other.dm [hunk 2]
```diff
@@ -973,6 +973,18 @@
...
+/datum/reagent/plasma/hemocatalene
+	name = "Hemocatalene Plasma"
+	id = PLASMA_HEMOCATALENE
+	description = "A red-ish plasma..."
+	color = "#cf5b51"
+	overdose = REAGENTS_OVERDOSE
+	overdose_critical = REAGENTS_OVERDOSE_CRITICAL
+	chemclass = CHEM_CLASS_SPECIAL
+	objective_value = OBJECTIVE_EXTREME_VALUE
+	properties = list(PROPERTY_HEMOGENIC = 20, PROPERTY_REPAIRING = 2, PROPERTY_TOXIC = 1)
```
Comment (Unknownity) [inline code/modules/reagents/chemistry_reagents/other.dm:985]: "the latter providing the target with increased blood regeneration, **brute damage healing**, and toxin damage."
Repairing provides healing only to cybernetic limbs, not brute in general. What you're looking for is Neogenetic (NGN)

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/carrier/reaper.dm [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (ThePiachu) [inline code/modules/mob/living/carbon/xenomorph/mutators/strains/carrier/reaper.dm:1]: Empty file?

File: ColonialMarinesALPHA.dme [hunk 4]
```diff
@@ -510,19 +510,19 @@
...
-#include "code\datums\supply_packs\reagent_tanks.dm"
-#include "code\datums\supply_packs\research.dm"
-#include "code\datums\supply_packs\vehicle_ammo.dm"
 #include "code\datums\supply_packs\medical.dm"
 #include "code\datums\supply_packs\misc.dm"
+#include "code\datums\supply_packs\mortar.dm"
 #include "code\datums\supply_packs\operations.dm"
+#include "code\datums\supply_packs\reagent_tanks.dm"
+#include "code\datums\supply_packs\research.dm"
+#include "code\datums\supply_packs\restricted_equipement.dm"
```
Comment (ThePiachu) [inline ColonialMarinesALPHA.dme:522]: Rebase since we fixed the DME with #1882

**Review by ThePiachu — CHANGES_REQUESTED** [review]

**Comment by Jeff-jefferson-jeffery** [comment]
Our scientists are hard at work figuring out how to fuck up CM in the funniest possible ways.
First, we had agents lobby for removal of groundside surgery. We set the stage early. We didn't want to be detected. After all, the less enjoyment inclined have always worked against ease of treatment. It wouldn't register of any radars.
Those got merged, so we moved onto stage two. It took three weeks for the think-tank to figure it out.
A T3 xeno... like a carrier, but it can just infect you. It just hits you, and you're infected. Genius, right? We had it aaaalll figured out. $20 sent to an unscrupulous maintainer and it was TM'd. The plan was complete.
Maybe now I'd be able to win one round on CM.

**Pull request closed without merging**